space-framework/entry.sh

```diff
@@ -22,16 +22,31 @@ function echoerr() {
    echo "ERROR: $1" >&2
 }
 
+function set_config_and_crds() {
+    KUBECONFIG=
+    kubectl config set-cluster space-mgt --server="https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}" --certificate-authority=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    kubectl config set-credentials space-mgt --token="$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)"
+    kubectl config set-context space-mgt --cluster=space-mgt --user=space-mgt 
+    kubectl config use-context space-mgt
+    KUBECONFIG=/home/spacecore/.kube/config
+
+    echo "Create a secret for the core cluster."
+    kubectl --kubeconfig /home/spacecore/.kube/config create secret generic corecluster --from-file=kubeconfig=/home/spacecore/.kube/config
```

Comment:
The pod crashes if the secret already exists. Please see how the `kubestellar` secret is created and prepend something like this:
```
    echo Attempting to delete core cluster secret...
    if ! kubectl --kubeconfig /home/spacecore/.kube/config delete secret corecluster ; then
        echo "Nothing to delete."
    fi
```